.github/workflows/docker.yml

```
@@ -12,8 +12,6 @@ on:
       - views/**
       - openapi/**
   push:
-    branches:
-      - main
     tags:
       - 'v*'
```

with this you don't need the `if contains -rc` 

```suggestion
      - 'v[0-9]+\.[0-9]+\.[0-9]+$'
```